packages/react/src/use-render/useRender.ts

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import type { ComponentRenderFn } from '../utils/types';
+import { useComponentRenderer } from '../utils/useComponentRenderer';
+import { defaultRenderFunctions } from '../utils/defaultRenderFunctions';
+import { CustomStyleHookMapping as StateDataAttributes } from '../utils/getStyleHookProps';
```

Comment:
```suggestion
import { CustomStyleHookMapping as StateAttributes } from '../utils/getStyleHookProps';
```

Reply:
Done. Also updated in the props table: https://github.com/mui/base-ui/pull/1418/files#diff-a4b963fd83e20c9225af018585e14e171301cda8819bf4a23096b15a66ecdcb6R38